charts/all/nfd/templates/nfd-nodefeaturediscovery.yaml

```diff
@@ -0,0 +1,20 @@
+﻿# Copyright (c) 2022 - 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+---
+apiVersion: nfd.openshift.io/v1
+kind: NodeFeatureDiscovery
+metadata:
+  name: nfd-instance
+  namespace: openshift-nfd
+spec:
+  extraLabelNs:
+    - sgx.intel.com
+  resourceLabels:
+    - sgx.intel.com/epc
+  operand:
+    image: quay.io/openshift/origin-node-feature-discovery:4.13
```

Comment:
Should the version number here be parameterized?

Reply:
It is parametrized now.